_posts/2024-08-19-neural-sparse-v2-models.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title:  Improving search efficiency and accuracy with the newest v2 neural sparse models
+authors:
+  - zhichaog
+  - congguan
+  - yych
+  - dylantong
+  - kolchfa
+date: 2024-08-19
+categories:
+    - technical-posts
+has_science_table: true
+meta_keywords: OpenSearch semantic search, neural sparse search, semantic sparse retrieval
+meta_description: Accelerating inference and improving search with v2 neural sparse encoding models
```

Comment:
Please update the blog with the following meta: 

meta_keywords: neural sparse models, OpenSearch semantic search,  semantic sparse retrieval, neural search

meta_description: OpenSearch announces the availability of v2 series neural sparse models that enhance the efficiency of semantic sparse retrieval while accelerating inference and improving search.

Reply:
Done